cookbooks/aws-parallelcluster-environment/resources/efs/partial/_mount_umount.rb

```diff
@@ -43,14 +43,18 @@
 
     # See reference of mount options: https://docs.aws.amazon.com/efs/latest/ug/automount-with-efs-mount-helper.html
     mount_options = "_netdev,noresvport"
-    if efs_access_point_id
-      mount_options = "iam,tls,access_point=#{efs_access_point_id}"
-    elsif efs_encryption_in_transit == "true"
+    if efs_encryption_in_transit == "true"
       mount_options += ",tls"
+      # iam authorization requires tls
       if efs_iam_authorization == "true"
         mount_options += ",iam"
       end
+      # accesspoint requires tls
+      if efs_access_point_id
+        mount_options += ",accesspoint=#{efs_access_point_id},"
```

Comment:
following the pattern, this probably shouldn't have a comma at the end

Reply:
You are absolutely right. Was smuggled in by some internal merging on my side 👍